src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorExtensionTest.php

```diff
@@ -46,4 +54,67 @@ public function test2Dot5ValidationApi()
         $this->assertSame(TraversalStrategy::NONE, $metadata->traversalStrategy);
         $this->assertCount(0, $metadata->getPropertyMetadata('children'));
     }
+
+    /**
+     * @group legacy
+     */
+    public function testLegacyWithBadFormRendererType()
```

Comment:
This is not the legacy signature, but the new one though, as you don't pass the boolean.

Reply:
You are right. But when type hint will be set in 7.0. Args checking will be drop and tests must also be dropped.

I can keep `@group legacy` and only remove `legacy` in function name ?